### DISCUSSION TITLE
Selection template cassiopeia for offline page in the frontend

### DISCUSSION THREAD

**Discussion by WM-Loose** [templaza/astroid-framework#671] [Ideas]
Hello Sonny,
many customers would prefer to use the original Cassiopeia template as an offline site (for maintenance work, for example). We use a simple plugin that allows you to specify which template can be used for the offline site. 
Here is the link for that plugin: https://github.com/GHSVS-de/plg_system_offlineghsvs/releases/tag/2023.11.03. 
Could you also install something like this in the Astroid plugin? 
Then you could do without this additional plugin.

Regards
Dirk

**Comment by Banuk1616** [comment]
I had a look at the Cassiopeia offline site and I have to say that I also find the Cassiopeia offline site a little bit nicer. Perhaps it would be an improvement if you could not only integrate this plugin into Astroid but also design your own offline pages. You could then tell the visitors why the website is offline and for how long and design it nicely yourself. Kind regards Dirk

**Reply by WM-Loose** [reply]
Take a look at the setting in the options (Astroid template settings), coming soon. That's a possibility but not quite what I actually want with the offline site. You should simply be able to select the offline template in the plugin. Just like in the plugin I linked.
